gammapy/maps/region.py

```diff
@@ -183,6 +183,48 @@ def bin_volume(self):
 
         return bin_volume
 
+    def to_wcs_geom(self):
+        """Get the minimal equivalent geometry that
+            contains the region.
+
+        Returns
+        -------
+        wcs_geom : `~WcsGeom`
+            A WCS geometry object.
+        """
+        wcs_new = self.wcs.copy()
```

Comment:
Now looking at the implementation, I realise that the simple solution that you proposed could work with a little modification as well. Let me try to explain:

I think for the `RegionGeom` we have two `wcs` definitions one that is passed on `__init__`, which is not "spatially confined" and then the one created here which is equivalent, but "spatially confined". You described this nicely as "minimal equivalent geometry that contains the region". The point is now that if we just go through `WcsGeom.create()` to create this, the reference point for the projection will be shifted (forced to the center) and thus the original `wcs` is possibly modified. I noticed that the same now happens in the current implementation as well.

However I think there is good alternative solution, if we just go through `.cutout()` here is a minimal example:
```python3
from gammapy.maps import WcsGeom, RegionGeom
geom = RegionGeom.create("icrs;circle(0, 0, 1)")

wcs_geom = WcsGeom(wcs=geom.wcs, npix=geom.wcs.array_shape)
print(wcs_geom)

cutout = wcs_geom.cutout(position=geom.center_skydir, width=geom.width)
print(cutout)
```
This will leave the original reference point untouched and still create a "confined" wcs.

Reply:
Thank you so much Axel, that is so much nicer - I didn't know this method existed. The implementation now is much simpler and also more correct I think.